canonical/snapcraft.io
# create /beta

## Done
- Create snapcraft.io/beta

## How to QA
- Go to https://snapcraft-io-4413.demos.haus/beta/snaps, should return a list of snaps for the logged in user. If user not logged in, should redirect to /login and redirect back to https://snapcraft-io-4413.demos.haus/beta/snaps
- Go to https://snapcraft-io-4413.demos.haus/beta/store.json
- Go to /beta/<package_name>/card.json e.g https://snapcraft-io-4413.demos.haus/beta/code/card.json
## Issue / Card https://warthogs.atlassian.net/browse/WD-4229
Fixes #

## Screenshots
